test_autofit/mapper/prior/test_limits.py

```diff
@@ -64,9 +64,14 @@ def test_vector_ignore_limits(model):
     )
 
 
-def test_log_uniform():
-    prior = af.LogUniformPrior()
-
+@pytest.mark.parametrize(
```

Comment:
You sure GaussianPriors which have optional upper and lower limits are being handled correctly too?

Reply:
The default I used for testing is Gaussian. I guess I could make the tests slightly more concise and explicit by rolling GaussianPrior into this test